test/integration/__tests__/live-api.js

```diff
@@ -405,7 +405,7 @@ describe('API integration', () => {
 
       const promise = createAssembly(client, opts)
       await promise.catch((err) => {
-        expect(err).toMatchObject({ transloaditErrorCode: 'INTERNAL_COMMAND_ERROR', assemblyId: expect.any(String) })
+        expect(err).toMatchObject({ transloaditErrorCode: 'INVALID_INPUT_ERROR', assemblyId: expect.any(String) })
```

Comment:
Shall we announce this? Since it's a breaking change and more people likely will notice

Reply:
Since we don't have a channel for dev updates I think that's the best place yes